container/shim/src/fetchers/lassie.js

```diff
@@ -297,6 +307,31 @@ async function getRequestedBlockFromCar(streamIn, streamOut, cidObj, filename) {
   }
 }
 
+function generatePeersLassieUrl(knownPeers) {
+  const peerUrls = [];
+  if (knownPeers) {
+    const knownPeerList = [];
+    Object.values(knownPeers).forEach((peerList) => {
+      knownPeerList.push(...peerList);
+    });
```

Comment:
nit: this can be simplified to
```suggestion
    const knownPeerList = Object.values(knownPeers).flat()
```